che-editors.yaml

```
@@ -0,0 +1,269 @@
+version: 1.0.0
+editors:
+  - id: eclipse/che-theia/next
+    endpoints:
+      - name: "theia"
+        public: true
+        targetPort: 3100
+        attributes:
+          protocol: http
+          type: ide
+          secure: true
+          cookiesAuthEnabled: true
+          discoverable: false
+      - name: "webviews"
+        public: true
+        targetPort: 3100
+        attributes:
+          protocol: http
+          type: webview
+          secure: true
+          cookiesAuthEnabled: true
+          discoverable: false
+          unique: true
+      - name: "mini-browser"
+        public: true
+        targetPort: 3100
+        attributes:
+          protocol: http
+          type: mini-browser
+          secure: true
+          cookiesAuthEnabled: true
+          discoverable: false
+          unique: true
+      - name: "theia-dev"
+        public: true
+        targetPort: 3130
+        attributes:
+          protocol: http
+          type: ide-dev
+          discoverable: false
+      - name: "theia-redirect-1"
+        public: true
+        targetPort: 13131
+        attributes:
+          protocol: http
+          discoverable: false
+      - name: "theia-redirect-2"
+        public: true
+        targetPort: 13132
+        attributes:
+          protocol: http
+          discoverable: false
+      - name: "theia-redirect-3"
+        public: true
+        targetPort: 13133
+        attributes:
+          protocol: http
+          discoverable: false
+    containers:
+      - name: theia-ide
+        image: "quay.io/eclipse/che-theia:next"
+        env:
+          - name: THEIA_PLUGINS
+            value: local-dir:///plugins
+          - name: HOSTED_PLUGIN_HOSTNAME
+            value: 0.0.0.0
+          - name: HOSTED_PLUGIN_PORT
+            value: "3130"
+          - name: THEIA_HOST
+            value: 127.0.0.1
+        volumes:
+          - name: plugins
+            path: "/plugins"
+        mountSources: true
+        memoryLimit: "512M"
+    initContainers:
+      - name: remote-runtime-injector
+        image: "quay.io/eclipse/che-theia-endpoint-runtime-binary:next"
+        volumes:
+          - name: remote-endpoint
+            path: "/remote-endpoint"
+            ephemeral: true
+        env:
+          - name: PLUGIN_REMOTE_ENDPOINT_EXECUTABLE
+            value: /remote-endpoint/plugin-remote-endpoint
+          - name: REMOTE_ENDPOINT_VOLUME_NAME
+            value: remote-endpoint
+  - id: eclipse/che-theia/7.22.2
```

let's make it to 7.23
```suggestion
  - id: eclipse/che-theia/7.23.0
```